ci/src/artifacts/upload.sh

```diff
@@ -53,7 +68,7 @@ echo "done uploading to AWS" >&2
 # Upload to Cloudflare's R2 (S3)
 # using profile 'cf' to look up the right creds in ~/.aws/credentials
 echo "uploading $f to Cloudflare" >&2
-echo AWS_PROFILE=cf "$RCLONE" -v \
+AWS_PROFILE=cf "$RCLONE" -v \
```

Comment:
looks like I might have disabled Cloudflare upload by mistake in https://github.com/dfinity/ic/pull/4393